src/support/import-supervisor.js

```diff
@@ -93,12 +94,13 @@ function ImportSupervisor(services, config) {
    * @param {object} options - Client provided options for the import job.
    * @returns {Promise<ImportJob>}
    */
-  async function createNewImportJob(urls, importQueueId, apiKey, options) {
+  async function createNewImportJob(urls, importQueueId, hashedApiKey, options) {
```

Comment:
Does the GSI we use to query the import-job table need to change?

Reply:
I think the GSI should be fine as we do not query the import-job table based on the hashedApiKey / the API key yet. But, we'll need to modify the import-jobs table to change the `apiKey` column attribute to `hashedApiKey`.